src/clr-angular/utils/focus/direction.enum.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum Direction {
```

Comment:
I would like a better name for this.

Reply:
NavigationDirection? KeyboardNavDirection? OneDirection?

Reply:
KeyboardNavDirection 👍 
OneDirection 😂

Reply:
I'm going with `ArrowKeyDirection`.

But `OneDirection` was a close second. 👍